packages/@sanity/base/tsconfig.dist.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../../tsconfig.dist",
+  "include": ["src"],
+  "compilerOptions": {
+    "outDir": "./lib",
+    "strict": false,
+    "strictNullChecks": false,
+    "jsx": "react",
+    "plugins": [
+      {
+        "name": "typescript-plugin-css-modules",
+        "options": {
+          "customMatcher": "\\.css$"
+        }
+      }
+    ],
+
+    "baseUrl": "./src",
+    "paths": {
+      "part:@sanity/base/datastore/history": ["./datastores/history"],
+      "part:@sanity/base/user": ["./datastores/user"]
```

Comment:
Should more of the datastore parts be aliased here? Or, what is the reason we only alias these?

Reply:
These are the parts which `@sanity/base` itself depends on. This could also be defined in `src/@types/parts.d.ts`, but it's less verbose when defined in tsconfig.json.